exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/FooterGatherer.java

```diff
@@ -66,8 +69,8 @@ private static void checkMagicBytes(FileStatus status, byte[] data, int offset)
   }
 
   public static List<Footer> getFooters(final Configuration conf, List<FileStatus> statuses, int parallelism) throws IOException {
-    final List<TimedRunnable<Footer>> readers = Lists.newArrayList();
-    List<Footer> foundFooters = Lists.newArrayList();
+    final List<TimedCallable<Footer>> readers = new ArrayList<>();
+    final List<Footer> foundFooters = new ArrayList<>();
```

Comment:
Any specific reason to not use Lists.newArrayList?

Reply:
Please see Guava Java doc [Lists.newArrayList()](http://google.github.io/guava/releases/snapshot/api/docs/com/google/common/collect/Lists.html#newArrayList--):

> **Note for Java 7 and later:** this method is now unnecessary and should be treated as deprecated. Instead, use the ArrayList [constructor](https://docs.oracle.com/javase/9/docs/api/java/util/ArrayList.html?is-external=true#ArrayList--) directly, taking advantage of the new ["diamond" syntax](http://goo.gl/iz2Wi).